tests/emailController.test.js

```diff
@@ -1,19 +1,19 @@
-import chai from 'chai';
-import chaiHttp from 'chai-http';
-import app from '..';
+// import chai from 'chai';
```

Comment:
#### Critical
Why do we have the entire test file commented out? If it's not needed then delete it but I'll like to know why this is commented out in the first place